public/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -0,0 +1,47 @@
+define(["jquery", "fc/bramble-keyhandler", "fc/bramble-underlay"], function($, KeyHandler, Underlay) {
+
+  function PopupMenu(button, menu) {
+    var self = this;
+    self._button$ = $(button);
+    self._menu$ = $(menu);
+
+    // Toggle the menu on/off when the button is clicked.
+    self._button$.on("click", function() {
+      if(!self.showing) {
+        self.show();
+      } else {
+        self.close();
+      }
+    });
+  }
+  PopupMenu.create = function(button, menu) {
+    return new PopupMenu(button, menu);
```

Comment:
It's mostly to keep jshint happy, so that I don't have a variable I never use in the caller (i.e., I can ignore the return value if I don't use it).